go.mod

```diff
@@ -1,66 +1,23 @@
 module github.com/cymon1997/go-template
 
-go 1.18
+go 1.16
```

Comment:
why do we need to downgrade the go version?

Reply:
for development compatibility because most devs still use 1.16 as their default

Reply:
ok then